python/pyspark/sql/catalog.py

```diff
@@ -273,15 +273,15 @@ def databaseExists(self, dbName: str) -> bool:
 
         >>> spark.catalog.databaseExists("test_new_database")
         False
-        >>> _ = spark.sql("CREATE DATABASE test_new_database")
+        >>> _ = spark.sql("CREATE DATABASE test_new_database").collect()
```

Comment:
Hmmm ... this is orthogonal issue but just to make sure we don't forget. I think we should make `sql` method in Spark Connect to be analyzed per every call. Otherwise, we can't make it compatible with the existing PySpark usage.

I changes the existing doctest here for now to make the tests pass but should change this back ideally.

Reply:
cc @hvanhovell @grundprinzip FYI

Reply:
what about create a jira ticket for this (should change this back ideally)?

Reply:
👌